Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -280,16 +280,16 @@ private bool HandleTryPullObject(ICommonSession? session, EntityCoordinates coor
                 return true;
             }
 
-            if (userEntity == uid)
+            if (userEntity.Value == uid)
                 return false;
```

Comment:
My understanding was that if you have some combination of nullable or non-nullable structs, the equality operator will automatically check the actual struct values if the structs are both not-null. Is that not correct?

Reply:
No clue, although the reason I'm doing `userEntity.Value` here is because `ValidateClientInput` returned true, meaning that `userEntity` is not null